<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WEBCAM</class>
 <widget class="QMainWindow" name="WEBCAM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Rec</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="CorrectPatternHexa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Correción Patrón Hexagonal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CorrectBadPixels">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Correción Pixeles Muertos</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSeekThermalApp">
    <property name="title">
     <string>SeekThermalApp</string>
    </property>
   </widget>
   <addaction name="menuSeekThermalApp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
